arcaflow_plugin_stressng/stressng_schema.py

```diff
@@ -8,19 +8,17 @@
 from arcaflow_plugin_sdk import annotations
 
 
-def params_to_jobfile(params):
+def params_to_jobfile(params: dict) -> str:
```

Comment:
Not to be annoying, (and I'm not insisting on a change), but with your latest changes there no longer seems to be any need to check for `stressor` and `workers` since you're crafting the exact set of parameters in each call.

Reply:
Yep, I overlooked this. However, if the `to_jobfile` functions worked the way I wanted them to, without explicit `key: value` definitions and instead calling the class as a whole somehow, then I would still need this.